themes/origins/styles/review-card.scss

```
@@ -0,0 +1,35 @@
+.review-card {
+  padding: 1.5em;
+  display: grid;
+  text-align: center;
+  gap: var(--gap-xl);
+  border-radius: var(--radius-lg);
+  border: 1px solid color-mix(in srgb, currentColor 10%, transparent);
+
+  .review-rating {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    .stars {
+      padding: 5px 10px;
+      display: inline-flex;
+      gap: var(--gap-xs);
+      border-radius: var(--radius-xl);
+      color: var(--color-yellow-500);
+      background-color: color-mix(in srgb, var(--color-yellow-500) 10%, transparent);
```

Not sure why prettier didn't pick this up

```suggestion
      background-color: color-mix(
        in srgb,
        var(--color-yellow-500) 10%,
        transparent
      );
```